docs/DeveloperGuide.md

```diff
@@ -320,15 +320,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`   | User                 | View timing of upcoming events                   | I know when to go                                    |
 | `* *`   | User                 | See the most urgent events                       | I know what to do first                              |
 | `* *`   | Careless user        | Undo accidental deletion                         | I can undo deleted events                            |
-| `* *`   | Impatient person     | Search contacts                                  | I can quick find specific POC                        |
-| `* *`   | Impatient person     | Search events                                    | I can quick find specific event                      |
-| `* *`   | Lazy person          | Filter events to find similar events             | I can find similar events easily                     |
+| `* *`   | Impatient event     | Search contacts                                  | I can quick find specific POC                        |
```

Comment:
Maybe this is an accident made when renaming?

Reply:
Yeah, I'll fix it